src/URI/ByteString.hs

```diff
@@ -70,8 +70,6 @@ module URI.ByteString
     normalizeURIRef',
 
     -- * Low level utility functions
-    urlDecode,
```

Comment:
I'm curious if we really need to remove these from the public API. The PR description mentions not calling them internally anymore but I'm missing why these aren't provided to the end user.